### PR TITLE
Read AWS Secrets with 'SecretString'

### PR DESCRIPTION
**Pull request by nick-amplify**
Closes https://github.com/PrefectHQ/prefect-aws/issues/265

### Checklist

- [x] References any related issue by including "Closes #<Issue Number>" or "Closes <Issue URL>".
  - If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect-aws/issues/new/choose) first.
- [x] Includes tests or only affects documentation.
- [x] Passes `pre-commit` checks.
  - Run `pre-commit install && pre-commit run --all` locally for formatting and linting.
- [x] Summarizes PR's changes in [CHANGELOG.md](https://github.com/PrefectHQ/prefect-aws/blob/main/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 ### Changed
-
+* Added a check to AwsSecretsManager.load() for 'SecretString' as well as 'SecretBinary'
```
Comment (desertaxle) [inline CHANGELOG.md:13]:
```suggestion
- Added a check to AwsSecretsManager.load() for 'SecretString' as well as 'SecretBinary' - [#274](https://github.com/PrefectHQ/prefect-aws/pull/274)
```

**Review by desertaxle — CHANGES_REQUESTED** [review]
Thanks for contributing @nick-amplify! Could you please add some tests covering this new functionality?

**Comment by desertaxle** [comment]
@nick-amplify just checking to see if you can write some tests for this PR 🙂

**Comment by Bryan-Meier** [comment]
I'm curious if this PR will make it into the main branch any time soon? We are wanting to leverage the SecretString functionality as all of our Secrets are in SecretString format and not SecretBinary. Thanks in advance!

**Review by desertaxle — APPROVED** [review]

**Pull request merged**